### PR TITLE
Add com.mtkennerly.madamiru

### PR DESCRIPTION
**Pull request by mtkennerly**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly: Madamiru can play multiple videos, audio, and image files in a grid layout.
- [x] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:** https://github.com/mtkennerly/madamiru

I tested this on Ubuntu and made sure that videos, audio, and images all loaded correctly. I did see this printed in the console, but it seemed to be benign:

```
ubuntu@ubuntu:~/flathub$ flatpak run com.mtkennerly.madamiru ~/Downloads/test.madamiru
F: /run/user/1000/.flatpak/com.mtkennerly.madamiru/xdg-run/pulse is not a symlink to "../../flatpak/pulse" as expected: readlinkat: Invalid argument
```

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# [Madamiru](https://github.com/mtkennerly/madamiru)
+
+## Adding drives
```
Comment (hfiguiere) [inline README.md:3]: no user will ever see this.

**Comment by hfiguiere** [comment]
bot, build com.mtkennerly.madamiru

File: README.md (hunk 1 above)
Comment (mtkennerly) [reply README.md]: That's fair. I've removed it and just left a link to the original project :)

Comment (hfiguiere) [reply README.md]: I was just setting expectations. This kind of info can be useful for maintainers.  (and in that case explaining the existing permission instead)

**Pull request closed without merging**